README.md

```diff
@@ -80,6 +81,19 @@ All released versions can be found [here](https://github.com/sonatype-nexus-comm
 There are a range of options for installing the R plugin. You'll need to build it first, and
 then install the plugin with the options shown below:
 
+### Easiest Install
+
+Thanks to some upstream work in Nexus Repository (versions newer than 3.15), it's become a LOT easier to install a plugin. To install the `R` plugin, follow these steps:
+
+* Clone this repo and `cd` to the cloned directory location
+* Build the plugin with `mvn clean package -PbuildKar`
+* Copy the `nexus-repository-r-1.0.4-bundle.kar` file from your `target` folder to the `deploy` folder for your Nexus Repository installation.
```

Comment:
Q: Does "installation" infer $install-dir (vs $data-dir)?  I (minorly) feel like it'd be slightly more helpful to say rather than make them guess.

Reply:
I had the same questions when going through the process myself -- I was using the coda `easiest installation` instructions. I'll make an update with both your suggestions before the end of the day!

Reply:
Looks like it from `into <nexus_dir>/deploy` from "(more) Permanent Install"